helm/ingress-controller/templates/credentials-secret.yaml

```diff
@@ -1,10 +1,12 @@
-{{- if or (not (empty .Values.apiKey)) (not (empty .Values.authtoken)) }}
+{{- if eq .Values.credentials.secret.create true }}
```

Comment:
thinking about this a little bit more, I probably want to do a requires for this create value if either of the 2 credentials are set so the user gets a really obvious error that you passed in credentials but didn't specify create to avoid the footgun of having this just do nothing